html_css/css/style.css

```diff
@@ -1027,47 +1124,55 @@ button:active {
   cursor: pointer;
 }
 
-.blog__container_scroll-btn:hover {
+.blog__slider_scroll-btn:hover {
   box-shadow: 0px 4px 16px var(--arrow-box-shadow);
 }
 
-.blog__container_scroll-btn:active {
+.blog__slider_scroll-btn:active {
   border: 1px solid var(--arrow-play-button-hover);
 }
 
-.form {
+.registration-form {
+  padding: 78px 302px 0px 302px;
+}
+
+.registration-form form {
   margin: 0 auto;
   max-width: 450px;
 }
 
-.form input {
+.registration-form input {
```

Comment:
In code below you used this .registration-form input structure a few times. Good example why use class is better

Reply:
assigned new class